### ISSUE TITLE
Sticky cursor while moving window, bug or feature?

### ISSUE THREAD

**Issue by kaymio** [linuxmint/Cinnamon#7560]
```
 * Cinnamon version 3.8.1 but seen since a very long time
 * Distribution - Arch
 * RX480 - mesa 18.0.2-1
 * 64 bit
 ```

**Issue**
When dragging a window, mostly with mpv videos but also with others, the window sticks to the cursor and drops after a click is performed.

**Steps to reproduce**
It happens occasionally. It is not clear to me which circumstances lead to this behaviour. A gesture? A triple click? It happens most often when minimizing the mpv video player. In most cases the video window snaps back in its previous position, but sometimes it sticks to the cursor.

**Expected behaviour**
Windows not sticking to cursor.

**Other information**

**Comment by kaymio** [comment]
The only reference to a similar problem is from a W7 forum, which says this "feature", called ClickLock, can be disabled in W7.
https://www.sevenforums.com/general-discussion/32988-mouse-pointer-sticks-objects-has-clicked.html

When searching for Cinnamon or GNOME I always get sites about the lock screen.

**Comment by icarter09** [comment]
@kaymio are you still experiencing this issue?

**Comment by kaymio** [comment]
@icarter09 Recently I've been using [sway](https://github.com/swaywm/sway) while watching videos. Cinnamon is my fallback for gaming and other programs that don't like wayland/Xwayland. When I used Cinnamon this problem wasn't apparent any more. On the otherhand, mpv moved forward and I'm using a Wayland backend now. 
Let me watch it for a week and I'll tell whether it is still present or not.

**Comment by kaymio** [comment]
Yes, it was present today, once. It surely isn't as frequent as it has been, but still present.
